docs/chapters/04_features.md

```diff
@@ -773,6 +773,36 @@ export class GetProductsCount {
 
 
 
+#### Using pagination
+
+The current Booster GraphQL API includes an alternative type for your read models that stands for `List{"your-read-model-name"}`, which includes the functionality to work with pagination.
+
+The Read Model List type includes some new parameters that can be used on queries:
+ * `limit`; an integer that specifies the maximum number of items to be returned.
+ * `afterCursor`; an object that includes the key of the last retrieved item, so the next query will return the next page of results.
+
+ Example:
+```graphql
+query {
+  ListProductReadModels
+  (
+    limit: 1,
+    afterCursor: { id: "last-page-item"}
+  ) {
+    id
+    sku
+    availability
+    price
+  }
+}
+```
+
+Besides the parameters, this new type also returns a different result than the regular one. Called `{your-read-model-name}Connection`, it includes the following properties:
+ * `afterCursor`; if there are more results to paginate, it will return the object to pass to the `cursor` parameter on the next query. If there aren't more items to be shown, it will be undefined.
```

Comment:
🤔 Isn't this inverted? The result returns `cursor`, not "afterCursor", and it should say "...  the object to pass to the `afterCursor` parameter in the next query..."

Reply:
You're right!